data/com/mojang/blaze3d/vertex/VertexMultiConsumer.mapping

```diff
@@ -51,4 +51,20 @@ CLASS com/mojang/blaze3d/vertex/VertexMultiConsumer
 		METHOD <init> ([Lcom/mojang/blaze3d/vertex/VertexConsumer;)V
 			ARG 1 delegates
 		METHOD forEach (Ljava/util/function/Consumer;)V
-			ARG 1 consumerConsumer
+			ARG 1 metaConsumer
```

Comment:
I've brought this up in the internal channel, but we should also consider simply `consumer`, to bring it in line with [`Iterable#forEach(Consumer)`](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/lang/Iterable.html#forEach(java.util.function.Consumer)).